yt_dlp/extractor/xvideos.py

```diff
@@ -90,6 +90,51 @@ class XVideosIE(InfoExtractor):
     }, {
         'url': 'https://de.xvideos.com/video4588838/biker_takes_his_girl',
         'only_matching': True
+    }, {
+        'url': 'https://flashservice.xvideos.com/embedframe/ucuvbkfda4e',
+        'only_matching': True,
+    }, {
+        'url': 'https://www.xvideos.com/embedframe/ucuvbkfda4e',
+        'only_matching': True,
+    }, {
+        'url': 'http://static-hw.xvideos.com/swf/xv-player.swf?id_video=ucuvbkfda4e',
+        'only_matching': True,
+    }, {
+        'url': 'http://xvideos.com/video.ucuvbkfda4e/a_beautiful_red-haired_stranger_was_refused_but_still_came_to_my_room_for_sex',
+        'only_matching': True
+    }, {
+        'url': 'https://xvideos.com/video.ucuvbkfda4e/a_beautiful_red-haired_stranger_was_refused_but_still_came_to_my_room_for_sex',
+        'only_matching': True
+    }, {
+        'url': 'https://xvideos.es/video.ucuvbkfda4e/a_beautiful_red-haired_stranger_was_refused_but_still_came_to_my_room_for_sex',
+        'only_matching': True
+    }, {
+        'url': 'https://www.xvideos.es/video.ucuvbkfda4e/a_beautiful_red-haired_stranger_was_refused_but_still_came_to_my_room_for_sex',
+        'only_matching': True
+    }, {
+        'url': 'http://xvideos.es/video.ucuvbkfda4e/a_beautiful_red-haired_stranger_was_refused_but_still_came_to_my_room_for_sex',
+        'only_matching': True
+    }, {
+        'url': 'http://www.xvideos.es/video.ucuvbkfda4e/a_beautiful_red-haired_stranger_was_refused_but_still_came_to_my_room_for_sex',
+        'only_matching': True
+    }, {
+        'url': 'http://fr.xvideos.com/video.ucuvbkfda4e/a_beautiful_red-haired_stranger_was_refused_but_still_came_to_my_room_for_sex',
+        'only_matching': True
+    }, {
+        'url': 'https://fr.xvideos.com/video.ucuvbkfda4e/a_beautiful_red-haired_stranger_was_refused_but_still_came_to_my_room_for_sex',
+        'only_matching': True
+    }, {
+        'url': 'http://it.xvideos.com/video.ucuvbkfda4e/a_beautiful_red-haired_stranger_was_refused_but_still_came_to_my_room_for_sex',
+        'only_matching': True
+    }, {
+        'url': 'https://it.xvideos.com/video.ucuvbkfda4e/a_beautiful_red-haired_stranger_was_refused_but_still_came_to_my_room_for_sex',
+        'only_matching': True
+    }, {
+        'url': 'http://de.xvideos.com/video.ucuvbkfda4e/a_beautiful_red-haired_stranger_was_refused_but_still_came_to_my_room_for_sex',
+        'only_matching': True
+    }, {
+        'url': 'https://de.xvideos.com/video.ucuvbkfda4e/a_beautiful_red-haired_stranger_was_refused_but_still_came_to_my_room_for_sex',
+        'only_matching': True
```

Comment:
we really only need an .es test out of these

Reply:
Just copied the previous test. If the others are not needed, I can delete them.

Reply:
remove the others, except .es